Scripts/Runtime/Logging/BurstableLogger.cs

```diff
@@ -46,6 +52,7 @@ namespace Anvil.Unity.DOTS.Logging
         [NotBurstCompatible]
         public BurstableLogger(Log.Logger logger, string appendToMessagePrefix)
         {
+            m_ThreadIndex = UNSET_THREAD_INDEX;
```

Comment:
This is probably just being super pedantic but I'll mention it anyway. 

We should probably have line 39 set the thread index.

`private readonly int m_ThreadIndex = UNSET_THREAD_INDEX;`

Today, it will only get set to the `UNSET_THREAD_INDEX` if you use the specific constructors. 

Any `BurstableLogger` that is `default` or created in bulk for an array will use the default constructor and the thread index will be 0. 

The odds are super low this would happen and if it did then you miss out on your message prefix anyway but still, could be a very random weird bug in a specific case where you wanted it to run on the main thread without a prefix and now your log is telling you that you're off thread on thread 0. 

Bonus, you don't have to assign the `m_ThreadIndex` on line 80 anymore either.